.github/workflows/hls.yml

```diff
@@ -71,46 +31,11 @@ jobs:
       with:
         fetch-depth: '0'
 
-    # --------
-    # Install dependencies
-    # --------
-
     - name: "Install LLVM, Clang, MLIR, and Ninja"
       uses: ./.github/actions/InstallLlvmDependencies
```

Comment:
Do these lines still need to be here? Are the dependencies not installed in the BuildCirct composite action?

Reply:
The dependencies are only installed if you miss in the cache for the build CIRCT action.
See comment below.

Reply:
I missed that the first time. Thanks.